internal/lagoon/compose_test.go

```diff
@@ -82,15 +82,15 @@ func TestUnmarshaDockerComposeYAML(t *testing.T) {
 			wantOrder: []OriginalServiceOrder{
 				{Index: 0, Name: "cli"},
 				{Index: 1, Name: "nginx"},
 				{Index: 2, Name: "php"},
 				{Index: 3, Name: "mariadb"},
 				{Index: 4, Name: "redis"},
 				{Index: 5, Name: "chrome"},
-				{Index: 6, Name: "wait_dependencies"},
+				{Index: 6, Name: "wait-dependencies"},
```

Comment:
This one is going to be interesting. You had to change this in the test to fix the tests. But this is valid docker-compose and this particular test case is sanitized from a customer. So their builds will now fail.